lib/docsparser.js

```diff
@@ -16,6 +16,11 @@ DocsParser.prototype.init = function(viewSettings) {
 };
 
 DocsParser.prototype.is_existing_comment = function(line) {
+    // handle ES2015 generator shorhand for Object
+    // example: * funcname(var) {}
+    if (line.search(/^\s*\*\s?[\$\_A-Za-z][\$\_\w]+\s?\([\$\_A-Za-z][\$\_\w]+\)\s?\{/) >= 0) {
```

Comment:
* Please move this check in the JavaScript parser by creating a language specific `is_existing_comment` method there. Also please add tests for the new method and new behavior.
* Use `/yourregex/.test(line)` instead of `line.search(/yourregex/) >= 0` for simplicity.
* Please replace `\s?` with `\s*` since more then just one white space should be allowed.
* In regexes inside `[` `]` you don't need to escape characters like `_` and `$`.
* `\w` already includes underscores, please don't extra list them.
* Currently the regex only works for functions with one argument. To support multiple arguments, default arguments, object and array destructing, and the rest operator simply use `.*?`. This is also how its done [in the JavaScript parser](https://github.com/nikhilkalige/docblockr/blob/master/lib/languages/javascript.js#L62).